base_field_validator/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+Fields Validator
```

Comment:
make the title a real title by adding a lot of '===' above the title https://github.com/OCA/maintainer-tools/commit/d51c38f6b269d2d633d041109d04ee60992dafd8